.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ ubuntu-latest, macos-latest, windows-latest ]
-        go: [ '1.14', '1.15' ]
+        go: [ '1.14', '1.15', '1.16' ]
```

Comment:
```suggestion
        go: [ '1.15', '1.16' ]
```

Reply:
Thanks for checking on that. Any idea why it still compiles on 1.14 though? :thinking: shouldn't it fail to compile if we use new stuff like that? Maybe I misunderstand how the Go compiler works.

Reply:
Ha, good question. Now I see the trick. In go1.14, the value of `ok` will be `false`:

```go
comparer, ok := peerCert.PublicKey.(interface{ Equal(crypto.PublicKey) bool }) 
 if !ok || !comparer.Equal(allowedKey) { 
    continue
```

This means in go1.14, the loop will hit `continue` at every iteration, and the entire func ends up returning APIError:

https://github.com/caddyserver/caddy/blob/ec3ac840cf09ec43b9130b41a46c07827b2ff0b6/admin.go#L597-L603

It means users lose an entire feature by compiling with go1.14. The type-assertion is fail-safe so it **will not panic** if compiled with < go1.15, but I'd expect confused users hitting the forum.

Reply:
Okay @mholt are we pulling the trigger on the bump to `go1.15` then?

Reply:
Ah yep, I meant to say this, sorry. We need to bump the minimum to Go 1.15 for the 2.4 tree.